ui-core/src/styles/select.css

```diff
@@ -1,15 +1,19 @@
-.custom-select {
+.osrd-ui-custom-select {
   border-radius: 4px;
   border: 1px solid rgba(148, 145, 142, 1);
   box-shadow:
     0px 1px 1px rgba(0, 0, 0, 0.2),
     0px 2px 4px rgba(0, 0, 0, 0.1);
   height: 2.5rem;
-  padding: 0.438rem 0.813rem 0.562rem 0.813rem;
+  padding: 0.438rem 2rem 0.563rem 0.813rem;
   letter-spacing: 0;
   text-align: left;
   line-height: 1.5rem;
   width: 100%;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: inline('../assets/triangle-up-down.svg') no-repeat right .75rem center;
```

Comment:
This works fine but it's not what we use in the rest of the app. I'm afraid it might be difficult if we need to sometime hide it, focus it for accessibility etc. As discussed, I would like to discuss it together to understand what are the advantages :)